scripts/interop_start.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+"""
+2019/09/08 -- Interop start script.
+This script is intended for dockerfile deployment for interop testing.
+This script is fragile and subject to break as flags change.
+Use at your own risk!
+"""
+
+# Flags
+IDENTITY="" # P2P private key
+PEERS="" # Comma separated list of peers
+NUM_VALIDATORS="3" # Positive number of validators to operate.
+GEN_STATE="" # filepath to ssz encoded state.
+PORT="8000" # port to serve p2p traffic
+YAML_KEY_FILE="/launch/keys.yaml" # Path to yaml keyfile as defined here: https://github.com/ethereum/eth2.0-pm/tree/master/interop/mocked_start
+
+# Constants
+BEACON_LOG_FILE="/tmp/beacon.log"
+VALIDATOR_LOG_FILE="/tmp/validator.log"
+
+usage() {
+    echo "--identity=<identity>"
+    echo "--peer=<peer>"
+    echo "--num-validators=<number>"
+    echo "--gen-state=<file path>"
+    port "--port=<port number>"
+}
+
+while [ "$1" != "" ];
+do
+    PARAM=`echo $1 | awk -F= '{print $1}'`
+    VALUE=`echo $1 | sed 's/^[^=]*=//g'`
+
+    case $PARAM in
+        --identity)
+            IDENTITY=$VALUE
+            ;;
+        --peers)
+            PEERS+=",$VALUE"
+            ;;
+        --validator-keys)
```

Comment:
@renaynay please take a look, is this the correct flag name? They aren't defined in the google doc

Reply:
@prestonvanloon pls take a look here: https://github.com/whiteblock/dockerfiles/blob/new-prysm/ethereum/prysm/start.sh#L26

Just updated it 20 min ago. This will ask for a flag before adding each individual peer like 

`--peer=<peer> --peer=<additional peer> --peer=<additional peer>` and so forth

also NUM_VALIDATORS --> VALIDATOR_KEYS (takes a file path to /launch/keys.yaml) (fyi)